
object_tracker_controller_service_name: object_tracker_controller_service
object_tracker_service_name: multi_object_tracker_service
object_finder_service_name: object_finder_service

objects:
  # ros package name which contains the object models
  package: scooping_cv

  # directory to the object models within the package
  directory: meshes/tracking


